Source/WPEFramework/Config.h

```diff
@@ -357,6 +357,7 @@ namespace PluginHost {
 #endif
                 , ProxyStubPath()
                 , PostMortemPath(_T("/opt/minidumps"))
+                , MessageControlPath(_T("/tmp/MessageDispatcher"))
```

Comment:
The configurable part of the path should be relative to the volatile path, not an absolute path